Source/JavaScriptCore/b3/air/AirAllocateRegistersAndStackAndGenerateCode.cpp

```diff
@@ -420,7 +428,9 @@ void GenerateAndAllocateRegisters::prepareForGeneration()
         for (Reg reg : m_registers[bank]) {
             m_allowedRegisters.add(reg, IgnoreVectors);
             TmpData& data = m_map[Tmp(reg)];
-            unsigned slotSize = conservativeRegisterBytesWithoutVectors(bank);
+            unsigned slotSize = conservativeRegisterBytes(bank);
+            if (!Options::useWebAssemblySIMD())
+                slotSize = conservativeRegisterBytesWithoutVectors(bank);
             data.spillSlot = m_code.addStackSlot(slotSize, StackSlotKind::Spill);
             dataLogLnIf(GenerateAndAllocateRegistersInternal::verbose, "allowedRegisters: reg: ", reg, " -> slot ", data.spillSlot);
             data.reg = Reg();
```

Comment:
L445 is using `conservativeRegisterBytesWithoutVectors`, is it right?

Reply:
Yes. That's the smallest a slot can be (8 bytes even on 32-bit), but the slot->byteSize() can be bigger